docs/book/v2/getting-started/skeleton.md

```diff
@@ -22,13 +22,14 @@ This will prompt you to choose:
   templates, configuration, assets, etc.).
 
 - A dependency injection container. We recommend using the default, Zend
-  ServiceManager.
+  ServiceManager. (We use ServiceManager in the examples below, so 
+  choose it if you want to code along).
```

Comment:
This can be left out. The command below to generate  factory should be changed (more below).